### PR TITLE
Add default values for cashier request

### PR DESCRIPTION
**Pull request by lexerom**
## Summary

Use default values, if amounts and/or cashier amount are empty and there is no matching strategies.

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by swilczynski — DISMISSED** [review]

File: openapi/components/requestBodies/CashierRequest.yaml [hunk 1]
```diff
@@ -15,6 +15,19 @@ properties:
     description: |-
```
Comment (justnero) [inline openapi/components/requestBodies/CashierRequest.yaml:15]: Please, rename this file to `PostCachierRequest` as it causes non-ideal name resolution for PHP SDK at https://github.com/Rebilly/rebilly-php/pull/652/files#diff-0b802e2dec095bbc6dbcbf0365a4e379b929d34e7b81132aba531d76c42fd135

**Review by justnero — DISMISSED** [review]

File: openapi/components/requestBodies/PostCashierRequest.yaml [hunk 2]
```diff
@@ -15,6 +15,19 @@ properties:
     description: |-
       ID of a strategy to be applied for this request for `amounts` and `customAmount`.
       If this field is not specified, a randomly selected strategy with a matching `filter` value is applied for empty `amounts` and `customAmount`.
+      If there is no matching strategies, default strategy with the following parameters is applied for empty `amounts` and `customAmount`:
```
Comment (ninjasimon) [inline openapi/components/requestBodies/PostCashierRequest.yaml:18]:
```suggestion
      If there is no matching strategy, the default strategy with the following parameters is applied for empty `amounts` and `customAmount`:
```
Small improvement suggestion.

**Review by adamaltman — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Review by ninjasimon — APPROVED** [review]

**Review by ikram-patah — APPROVED** [review]

**Comment by ikram-patah** [comment]
Question: Would the following payload trigger this logic?
```
{
  "amounts": [],
  "strategyId": null,
}
```
emphasis on passing the `amounts` as `null` here.

**Comment by lexerom** [comment]
> Question: Would the following payload trigger this logic?
> 
> ```
> {
>   "amounts": [],
>   "strategyId": null,
> }
> ```
> 
> emphasis on passing the `amounts` as `null` here.

Yes, it will trigger default behavior.

**Pull request merged**